### PR TITLE
Add a fixed bound check to the classifier.

### PR DESCRIPTION
**Pull request by snim2**
Improve the classifier by checking against a fixed bound or a percentage bound, as appropriate.

### CODE REVIEW

File: bin/mark_changepoints_in_json [hunk 1]
```diff
@@ -85,28 +86,28 @@ class Segments(object):
...
-
-        upper_bound = 1.0 + self.half_bound
-        lower_bound = 1.0 - self.half_bound
+        lower_bound = min(self.segments[-1].mean * (1.0 - self.half_bound),
+                          self.segments[-1].mean - self.delta)
+        upper_bound = max(self.segments[-1].mean * (1.0 + self.half_bound),
+                          self.segments[-1].mean + self.delta)
         classification = 'flat'
-        last_segment = self.segments[-1]  # Hopefully the steady state segment.
+        # last_segment = self.segments[-1]  # Hopefully the steady state segment.
```
Comment (ltratt) [inline bin/mark_changepoints_in_json]: Commented code.

**Comment by ltratt** [comment]
One minor comment; let's fix that, then we can probably squash and merge.

**Comment by snim2** [comment]
Ok, good to squash?

**Comment by ltratt** [comment]
Yep, let's squash.

**Comment by snim2** [comment]
squashed

**Review by ltratt — APPROVED** [review]

**Pull request merged**